src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -28,6 +28,9 @@
     <ProjectJson>$(MsBuildThisFileDirectory)windows\project.json</ProjectJson>
     <ProjectLockJson>$(MsBuildThisFileDirectory)windows\project.lock.json</ProjectLockJson>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(IsNETNativeLibrary)' != 'true' AND '$(TargetsWindows)' == 'true'">
+    <DefineConstants>>$(DefineConstants);SUPPORTS_WINDOWSIDENTITY</DefineConstants>
+  </PropertyGroup>
```

Comment:
Nit: we originally chose "FEATURE" as the prefix for our conditional compiles, so FEATURE_WINDOWSIDENTITY might be more consistent.  Not required, just observing it.